docs/content/anonymization/performance.md

```diff
@@ -0,0 +1,184 @@
+    private ?int $id = null;
+
+    #[ORM\ManyToOne(inversedBy: 'addresses')]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?Customer $customer = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $street = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $zipCode = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $city = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $country = null;
+
+    //...
+}
+```
+
+```php [Order]
+#[ORM\Entity(repositoryClass: OrderRepository::class)]
+#[ORM\Table(name: '`order`')]
+class Order
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\ManyToOne(inversedBy: 'orders')]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?Customer $customer = null;
+
+    #[ORM\Column(length: 255)]
+    #[Anonymize(type: 'fr-fr.phone')]
+    private ?string $telephone = null;
+
+    #[ORM\Column(length: 255)]
+    #[Anonymize(type: 'email')]
+    private ?string $email = null;
+
+    #[ORM\Column]
+    #[Anonymize(type: 'float', options: ['min' => 10, 'max' => 99])]
+    private ?float $amount = null;
+
+    #[ORM\Column]
+    private ?\DateTimeImmutable $createdAt = null;
+
+    #[ORM\ManyToOne]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?Address $billingAddress = null;
+
+    #[ORM\ManyToOne]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?Address $shippingAddress = null;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    #[Anonymize(type: 'lorem')]
+    private ?string $note = null;
+
+    //...
+}
+```
+:::
+
+Then, we executed the `db-tools:anonymize` command on different backups with different
+amount of data.
+
+Here are the results:
+
+| Customer            | Address      | Order               | PostgreSQL | SQLite | MariaDB  | MySQL
+|---------------------|--------------|---------------------|------------|--------|----------|-------
+| 100&nbsp;000        | -            | -                   | 5s         | 7s     | 20s      | 53s
+| 500&nbsp;000        | -            | -                   | 9s         | 10s    | 37s      | 3m 44s
+| 1&nbsp;000&nbsp;000 | -            | -                   | 16s        | 16s    | 1m 23s   | 36m 56s
+| -                   | 200&nbsp;000 | -                   | 6s         | 10s    | 26s      | 42s
+| -                   | -            | 1&nbsp;000&nbsp;000 | 16s        | 11s    | 1m 15s   | 25m 1s
+| 100&nbsp;000        | 200&nbsp;000 | -                   | 7s         | 10s    | 32s      | 1m 16s
+| 100&nbsp;000        | 200&nbsp;000 | 1&nbsp;000&nbsp;000 | 24s        | 25s    | 1m 40s   | 36m 47s
+
+<small>
+<strong>NB1</strong>: The values indicated are approximate and vary slightly each time the command is
+executed. A margin of error of 10% can be assumed.
+</small><br>
+<small>
+<strong>NB2</strong>: Each database vendor docker image has been used as is. Without any tweaking.
+This could explain the bad results for MySQL compare to MariaDB's ones.
```

Comment:
```suggestion
<strong>NB2</strong>: Each database vendor docker image has been used as it. Without any tweaking.
This could explain the bad results for MySQL compared to MariaDB's ones.
```

Reply:
```suggestion
<strong>NB2</strong>: Each database vendor docker image has been used as is. Without any tweaking.
This could explain the bad results for MySQL compared to MariaDB's ones.
```